Doc set for VMFR (#110937)
openshift/openshift-docs: backup_and_restore/application_backup_and_restore/oadp-vmfr/oadp-vmfr.adoc

:_mod-docs-content-type: ASSEMBLY
include::_attributes/common-attributes.adoc[]

[id="oadp-vmfr"]
= {oadp-short} virtual machine file restore
:context: oadp-vmfr

toc::[]

[role="_abstract"]

> Use {oadp-short} virtual machine file restore (VMFR) to recover individual files from `kubevirt` virtual machine (VM) backups without restoring the entire VM. This helps you to browse, compare, and download files from multiple backups simultaneously.

Reviewer: ```suggestion
Use {oadp-short} virtual machine file restore (VMFR) to recover individual files from `kubevirt` VM backups without restoring the entire VM. You can browse, compare, and download files from multiple backups simultaneously.
```
This helps you to browse..." can be simplified to "You can browse..." to make the text more active and focused on the user's capabilities.
